products/views.py

```diff
@@ -1 +1,229 @@
+                description = description,
+                dosage      = dosage,
+                manual      = manual
+                )[0]
+            
+            subcategory_objs = helper_add_many_to_many(subcategories, SubCategory)
+            skin_objs        = helper_add_many_to_many(skin_types, Skin)
+            feel_objs        = helper_add_many_to_many(feels, Feel)
+            ingredient_objs  = helper_add_many_to_many(ingredients, Ingredient)
+            texture_objs     = helper_add_many_to_many(textures, Texture)
+            aroma_objs       = helper_add_many_to_many(aromas, Aroma)
+
+            product.subcategories.add(*subcategory_objs)
+            product.skin_types.add(*skin_objs)
+            product.feels.add(*feel_objs)
+            product.ingredients.add(*ingredient_objs)
+            product.textures.add(*texture_objs)
+            product.aromas.add(*aroma_objs)
+
+            url_arr = media_urls.values()
+            for element in url_arr:
+                temp            = element.split("::")
+                url, media_type = temp[0], temp[1]
+                product.media.create(url=url, media_type=media_type)
+
+            product.save()
+
+            subcategory_objs[0].category = Category.objects.get(name=category)
+
+            return HttpResponse("PRODUCT_CREATED", status=201)
+            
+        except KeyError as e:
+            return HttpResponse(e, status=400)
+
+class ProductCategoryView(View):
+    def get(self, request):
+        try:
+            subcategories = SubCategory.objects.prefetch_related('products').all()
+
+            categories = [{
+                    "id"           : subcategory.category.id,
+                    "type"         : subcategory.category.name,
+                    "subcategories": [
+                        {
+                            "id"         : subcategory.id,
+                            "name"       : subcategory.name,
+                            "productList": [{
+                    "id"      : product.pk,
+                    "name"    : product.name,
+                    "capacity": product.size,
+                    "price"   : product.price,
+                } for product in subcategory.products.all()],
+                        }
+                    ]
+                } for subcategory in subcategories]
+            
+            return JsonResponse({
+                "total_cat_sub_prod": [
+                    Category.objects.all().count(), 
+                    subcategories.count(), 
+                    Product.objects.all().count()
+                    ],
+                "categories"        : categories
+                }, status=200)
+        except:
+            return HttpResponse("BAD_REQUEST", status=404)
+
+class ProductView(View):
+    def get_object(self, id):
+        try:
+            return Product.objects.get(pk=id)
+        except:
+            return HttpResponse("NOT_FOUND", stauts=404)
```

Comment:
```suggestion
    def get_object(self, id):
        try:
            return Product.objects.get(pk=id)
        except Product.DoesNotExist:
            return HttpResponse("NOT_FOUND", stauts=404)
```
데이터가 없는 경우에 발생하는 에러를 캐치하시고 싶은 것이라면 해당 에러를 명시해주셔야 합니다